package.json

```diff
@@ -19,7 +19,7 @@
     "jquery": "^3.5.1",
     "jquery-ui": "1.12.1",
     "jszip": "^3.4.0",
-    "kobo-common": "https://github.com/kobotoolbox/kobo-common.git#semver:1.28.0",
+    "kobo-common": "https://github.com/kobotoolbox/kobo-common.git#semver:1.33.0",
```

Comment:
[xref](https://github.com/kobotoolbox/kobo-common/commits/main): allow snake cases in enums